.travis.yml

```diff
@@ -6,15 +6,16 @@ php:
   - 5.6
   - 7.0
   - 7.1
+  - 7.2
 
 matrix:
   include:
     - php: 5.5
       env: dependencies=lowest
 
 before_script:
-  - composer install --no-interaction -o
-  - if [ "$dependencies" = "lowest" ]; then composer update -o --prefer-lowest; fi;
+  - composer update --no-interaction
```

Comment:
The point of committing the lock file is to have repeatable builds while testing. I think this should stay as `install`

Reply:
If using `install`, the package will have the dependencies problem because of the cache `composer.lock` file.
Please take a look at that previous [Travis build log](https://travis-ci.org/tebru/enum/builds/402562875).